test/Function/StackArgsWithFormals.js

```diff
@@ -0,0 +1,141 @@
+//Test1 : Tests Optimization with bailout.
+var shouldBailout = false;

```

Comment:
Discussed this in person - we should be tracing if the optimization is enabled or not. 
Also instead of printing values in the baseline, it is better if is expected and actual values comparisons in the test file itself and print only if that fails. 

---
In reply to: [67812487](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67812487) [](ancestors = 67812487,67765923)